File: C++ Solutions/NeitherMinimumnorMaximum2733.cpp
#include<iostream>
#include<vector>
#include<bits/stdc++.h>
using namespace std;


  int findNonMinOrMax(vector<int>& nums) {
    if (nums.size() < 3)
      return -1;
    sort(nums.begin(), nums.begin() + 3);
    return nums[1];
  }
int main(){
    vector<int> nums{3,2,1,4};
    findNonMinOrMax(nums);
    return 0;
}

File: JAVA Solutions/NeitherMinimumnorMaximum2733.java
import java.util.Arrays;

public class NeitherMinimumnorMaximum2733 {
    public static void main(String[] args) {
        int[] nums={3,2,1,4};
        System.out.println(findNonMinOrMax(nums));

    }
 
  static int findNonMinOrMax(int[] nums) {
    if (nums.length < 3)
      return -1;
    Arrays.sort(nums, 0, 3);
    return nums[1];
  }
}